main_COPY_AND_RENAME.py

```diff
@@ -29,7 +37,80 @@
         if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
             pygame.quit()
             sys.exit()
-        
+        # /--------------------------------/
+        # The first way of interacting with events
+        """"
+        elif event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_RIGHT and x < screen_width - width:
+                rect_x += rect_speed
+            elif event.key == pygame.K_LEFT and x > 0:
+                rect_x -= rect_speed
+            elif event.key == pygame.K_UP and y > 0:
+                rect_y -= rect_speed
+            elif event.key == pygame.K_DOWN and y < screen_height - height:
+                rect_y += rect_speed
+        elif event.type == pygame.MOUSEBUTTONDOWN:
+            if event.button == 1:
+                print("You pressed the left mouse botton!")
+                color = RED # Change colour to red
+            elif event.button == 3:
+                print("You pressed the right mouse button!")
+                color = GREEN # Change colour to green
+            elif event.button == 2:
+                # add your own code
+                print("You pressed the mouse wheel button!")
+            elif event.buttom == 4:
+                # add your own code
+                print("You used the mouse wheel scroll up!")
+            elif event.buttom == 5:
+                # add your own code
+                print("You used the mouse wheel scroll down!")
+            """
+
+    # /--------------------------------/
+    # The second way of interacting with events
+
+    keys = pygame.key.get_pressed()
```

Comment:
Consider renaming `keys` to `keys_pressed` to be more accurate
```suggestion
    keys_pressed = pygame.key.get_pressed()
```